pkg/operator/encryption/controllers/key_controller.go

```
@@ -220,10 +236,41 @@ func (c *keyController) checkAndCreateKeys(ctx context.Context, syncContext fact
+		return 0, nil
+	}
+
+	encryptionStatus, err := c.encryptionStatusProvider.GetKMSEncryptionStatus(ctx)
+	if err != nil {
+		return 0, fmt.Errorf("failed to get KMS encryption status for remote key rotation: %w", err)
+	}
+
+	return reconcileRemoteKeyRotation(ctx, c.secretClient, c.instanceName, snap.State.EncryptedGRs, snap.State.DesiredBeforePlan, encryptionStatus, systemRemoteKeyClock{})
+}
+
+func (c *keyController) checkAndCreateKeys(ctx context.Context, syncContext factory.SyncContext, encryptedGRs []schema.GroupResource) error {
```

golangci-lint (2.13.2)

[error] func (*keyController).checkAndCreateKeys is unused

(unused)